Libraries/Components/View/ViewAccessibility.d.ts

```diff
@@ -312,7 +332,6 @@ export type Role =
   | 'feed'
   | 'figure'
   | 'form'
-  | 'grid'
```

Comment:
Why did we get rid of `'grid'` here?

Reply:
Sorry. It was a mistake while removing the duplicate from RCTViewManager (related https://github.com/facebook/react-native/commit/f3d9f2ea233304870bd4ab67d9682af6eb0ae16f).
Fixed with commit https://github.com/facebook/react-native/pull/35130/commits/f4950258c789bd02694ff074a6414c821ed861a8

Reply:
As explained by Nick in https://github.com/facebook/react-native/pull/34477/files#r982953176, this roles are platform specific. I'm not adding them to the Role type.

Reply:
test cases included in https://github.com/facebook/react-native/pull/35130#issuecomment-1380190491

Reply:
related PR https://github.com/facebook/react-native-deprecated-modules/pull/15